examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  children: React.ReactNode;
+  title: string;
+};
+
+const getTabKey = (index: number, suffix?: string) =>
+  [`tab-${index}`, suffix].filter(Boolean).join('-');
+
+export function Tabs({ children }) {
+  const [currentTab, setCurrentTab] = useState(0);
+  const tabsRefs: HTMLElement[] = [];
+
+  const onKeyPress = (event: React.KeyboardEvent) => {
+    let tabIndex;
+    switch (event.key) {
+      case 'ArrowLeft':
+        tabIndex = Math.max(0, currentTab - 1);
+        break;
+      case 'ArrowRight':
+        tabIndex = Math.min(currentTab + 1, React.Children.count(children) - 1);
+        break;
+      default:
+        return;
+    }
+
+    tabsRefs[tabIndex].focus();
+    setCurrentTab(tabIndex);
```

Comment:
I'd use an effect to set the focus when the tab changes instead.

Reply:
you can't focus outside of an event handler, no?

Reply:
Can't you? https://codesandbox.io/s/misty-frost-ekigj?file=/src/App.js

Reply:
It does work on my side as well.